src/components/App.jsx

```diff
@@ -2,20 +2,19 @@ import React, { Component } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import DefaultLayout from '../layouts/default.layout.jsx';
 import MainLayout from '../layouts/main.layout.jsx';
-
+import { PrivateRoute } from '../components/auth/ProtectedRoute.jsx';
 class App extends Component {
 	render() {
 		return (
 			<div className='app'>
 				<Router>
 					<Switch>
-						<Route path='/' exact component={MainLayout} />
+						<PrivateRoute path='/' exact component={MainLayout} />
```

Comment:
@Jajabenit250 Why did you changed this to private route.

Reply:
@mujohn26 To check if the user is already signed in and got to dashboard if not he will go to login instead.